src/Uno.Extensions.Authentication.MSAL/MsalAuthenticationBuilderExtensions.cs

```diff
@@ -81,6 +86,9 @@ Action<StorageCreationPropertiesBuilder> store
 		string[] scopes
 		)
 	{
+#if !UNO_EXT_MSAL
+		return builder;
```

Comment:
Should it raise a warning somehow?

Reply:
I'm not sure how we should do that? I don't really have access to a logger here. Through an attribute maybe?